test/module/irohad/validation/chain_validation_test.cpp

```diff
@@ -47,7 +45,7 @@ class ChainValidationTest : public ::testing::Test {
             shared_model::interface::types::PubkeyType(std::string(32, '0'))));
     signatures.push_back(signature);
 
-    EXPECT_CALL(*block, height()).WillRepeatedly(Return(1));
+    EXPECT_CALL(*block, height()).WillRepeatedly(Return(height));
```

Comment:
So now `height == 2` but why it was changed?

Reply:
that is because now chain validator also checks the block height, but previously it used to check only previous hash. You may see that the checker function is invoked with a "previous ledger state" that now has the top height of 1:
https://github.com/hyperledger/iroha/blob/63e2ea2751f0a7a504158dbec878a0a295fa59aa/test/module/irohad/validation/chain_validation_test.cpp#L87-L89
so the new block itself must have height 2